pkg/rpc/rpc.go

```diff
@@ -45,10 +45,9 @@ type RPC struct {
 	lastRequestId   uint64
 	requestsMu      sync.Mutex
 
-	errChannel        chan struct{} // never transports a value, only closed through setErr() to signal an occurred error
-	err               *Error        // only initialized via setErr(), if a rpc (Fatal/non-recoverable) error has occurred
-	errOnce           sync.Once
-	requestedShutdown bool
```

Comment:
Please create a separate PR for removing that unused field so that we can make the CI happy again sooner.